packages/thirdweb/src/extensions/erc20/drops/write/claimTo.ts

```diff
@@ -34,6 +34,7 @@ export type ClaimToParams = {
  * ```
  * @throws If no claim condition is set
  * @returns A promise that resolves with the submitted transaction hash.
+ * @important If the claiming address is not the same as the receiving address, you need to specify the claimer address as the `from` param to avoid any issue with the allowlist (if one exists)
```

Comment:
I think we parse @note for the docs but not @important

Reply:
oh. thanks im gonna check the build for potal when I wake up

Reply:
We dont also render `@note`

Reply:
updated